tuxemon/core/item/conditions/type.py

```diff
@@ -48,8 +48,8 @@ class TypeCondition(ItemCondition):
     def test(self, target):
         ret = False
         if target.type1 is not None:
-            ret = any(target.type1 in p for p in self.parameters)
+            ret = any(target.type1.casefold() == p.casefold() for p in self.parameters)
```

Comment:
please use `lower` here because `casefold` is python3 only.  im not ready to remove python 2.7 support.